deploy-config-example.js

```diff
@@ -1,33 +1,35 @@
 var secrets = require('./secrets.js');
 
 var s3Credential = {
-  accessKeyId:     secrets.s3.accessKeyId,
+  accessKeyId: secrets.s3.accessKeyId,
   secretAccessKey: secrets.s3.secretAccessKey,
   params: {
-    Bucket:        'productboard-assets'
-  }
+    Bucket: 'productboard-assets',
+  },
 };
 
 var config = {
-
   redis: {
     development: {
       port: 6379,
       host: 'localhost',
       db: 1,
-      options: { },
+      options: {},
       indexPath: 'dist/index.html',
-      indexKey: 'app:%s',
-      metaKey: 'meta:%s',
+      indexKey: 'app:%s', // revision
+      branchKey: 'app:branch:%s', // branch name
+      branchRevKey: 'app:branch-revision:%s', // branch name
+      metaKey: 'meta:%s', // revision
+      mainBranchKey: 'app:current-branch',
       mainIndexKey: 'app:current',
       mainRevKey: 'app:current-revision',
-      revTimestampKey: 'app-timestamp:%s',
-      lastMajorTimestampKey: 'app:last-major-timestamp',
+      revTimestampKey: 'app-timestamp:%s', // revision
+      lastMajorTimestampKey: 'app:admin:last-major-timestamp:%s', // branch name
     },
     staging: {
-      port:    9999,
-      db:      0,
-      host:    'pub-redis-9999.us-east-1-1.2.ec2.garantiadata.com',
+      port: 9999,
+      db: 0,
+      host: 'pub-redis-9999.us-east-1-1.2.ec2.garantiadata.com',
```

Comment:
Anything that's not supposed to be in the repo can be in the `secrets.js` file. I'd leave it up to the library user which parts they consider to be a secret :)